lib/resources/r4/related_person_patch.yaml

```diff
@@ -88,6 +88,17 @@ operations:
                   "end": "2022-05-01T16:56:56.000Z"
                 },
                 "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/period"
+              },
+              {
+                'valueCodeableConcept': {
+                  'coding': [
+                    {
+                      'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/40',
+                      'code': '155'
+                    }
+                  ]
+                },
+                'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relation'
```

Comment:
Since we are adding the relation extension in the Add operation, we probably need to add the note for it?

Reply:
That's fine to add that here, but I left them out because I figured we could just add notes when we did the fhir.cerner.com for the standard code support.

Reply:
+1

Reply:
Since we are now doing the standard code support in the same PR,
should we now add the note to say:
`The relationship.relation extension must contain exactly one proprietary system or a standard system.`

Reply:
How about we follow something similar to what has been done for the appt resource. http://fhir.cerner.com/millennium/r4/scheduling/appointment/
"The relationship value.coding can be either from the standard or proprietary system, but not both."
"Only one coding may be provided in the value"

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/062a7f28bf665d671d55b5d50c1f8fa7067a2349#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR112-R113

Reply:
I apologize, but I see now that the 'value.coding' really only makes sense for the realtionship field in the patch operations. I think if we update the wording to something like:
"Each 'coding' can be either from the standard or proprietary system, but not both." 
"Only one coding may be provided for each element"
@exopsRamba What are your thoughts here?

Reply:
I think the first note looks good. For the second note `Only one coding may be provided for each element`, I believe we allow for multiple standard codes in a given coding. So it might be more accurate to say `If the coding is from a proprietary system, only one code maybe be provided` or something along those lines?

Reply:
+1 to Keifer's suggestion.

Reply:
Revision: "If the coding is from a proprietary system, only one code may be provided"

Reply:
Done where ever applied https://github.com/cerner/fhir.cerner.com/pull/662/commits/2ea0e3b1d34a19b00afe0d61be849cee9ee57764#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R150-R151

Reply:
Just FYI, 
when we provide 2 different standard codes in the body, fhir will not validate it but there will be an error at the local service end.
Local service will only allow multiple standard codes only when they are duplicate/same, and it will consider it to be a single code.

Considering that would the decision on the note still be fine?

@alexbeckner1 @exopsRamba

Reply:
@exopsRamba. Knowing that, does it make sense to use my pervious note? "Only one coding may be provided for each element."

Reply:
Yeah I think that would be best.

Reply:
Done made the change according to this comment everywhere https://github.com/cerner/fhir.cerner.com/pull/662#discussion_r652126559
change link: https://github.com/cerner/fhir.cerner.com/pull/662/commits/300867b2eb8e04f2456f4a36f5c45cc93172f23b#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR113

Reply:
I think the suggestion was to specify that they can only provide one coding regardless of whether it's a standard or proprietary system.

Reply:
Correct. We want to use something along the lines of my previous note "Only one coding may be provided in the value"

Reply:
Done as per the discussion offline https://github.com/cerner/fhir.cerner.com/pull/662/commits/efb7ac07367d7009d3785e8e9562c73fbb6f33b5#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R151